src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -71,4 +71,10 @@ public void addSections(long lineId, SectionRequest request) {
 
         line.validateAndAddSections(request.getDistance(), newUpStation, newDownStation);
     }
+
+    public void removeSectionByStationId(Long lineId, Long stationId) {
+        Line line = findByIdOrThrow(lineId);
+
+        line.validateAndRemoveSectionByStation(stationService.findById(stationId));
```

Comment:
```suggestion
        line.removeSectionByStation(stationService.findById(stationId));
```
이름은 이렇게만 해도 충분할 것 같아요! 이 메소드를 타고 들어가면 전부 validate가 prefix로 붙어있는데, 전부 제거해도 무방할 것 같습니다.

Reply:
음.. 저는 네이밍을 붙여주는게 내부 기능에 대해 명확할 것이라고 생각했는데, 그게 line의 역할은 아닌 것 같네요.! 수정했습니다.!